examples/run_distillation_math.py

```
@@ -0,0 +1,292 @@
+
+    # Load OpenMathInstruct2Dataset using nemo rl datasets
+    if data_config["dataset_name"] == "OpenMathInstruct-2":
+        print("Loading nvidia/OpenMathInstruct2Dataset for training and validation")
+        data: Any = OpenMathInstruct2Dataset(seed=seed)
+    elif data_config["dataset_name"] == "DeepScaler":
+        print(
+            "Loading agentica-org/DeepScaleR-Preview-Dataset for training and validation"
+        )
+        data: Any = DeepScalerDataset(seed=seed)
+    else:
+        raise ValueError(f"No processor for dataset {data_config['dataset_name']}.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)